huggingface/tensorflow/buildspec.yml

```diff
@@ -30,21 +41,39 @@ images:
     transformers_version: &TRANSFORMERS_VERSION 4.6.1
     datasets_version: &DATASETS_VERSION 1.6.2
     tag: !join [ *VERSION, '-', 'transformers', *TRANSFORMERS_VERSION, '-', *DEVICE_TYPE, '-', *TAG_PYTHON_VERSION, '-',
-                 *CUDA_VERSION, '-', *OS_VERSION ]
-    docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /,
-                         *CUDA_VERSION, /Dockerfile., *DEVICE_TYPE ]
+      *CUDA_VERSION, '-', *OS_VERSION ]
+    docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /, 
+      *CUDA_VERSION, /Dockerfile., *DEVICE_TYPE ]
+  BuildHuggingFaceTensorflowCPUPTInferencePy3DockerImage:
+    <<: *INFERENCE_REPOSITORY
+    build: &HUGGINGFACE_TENSORFLOW_CPU_INFERENCE_PY3 false
+    image_size_baseline: 4899
+    device_type: &DEVICE_TYPE cpu
+    python_version: &DOCKER_PYTHON_VERSION py3
+    tag_python_version: &TAG_PYTHON_VERSION py36
+    os_version: &OS_VERSION ubuntu18.04
+    transformers_version: &TRANSFORMERS_VERSION 4.6.0
+    inference_toolkit_version: &INFERENCE_TOOLKIT_VERSION 0.0.1.dev0
```

Comment:
Can this be set directly in the dockerfile?

The reason we define transformers_version in the buildspec is because it needs to be included in the image tag, and so it would be better to track it in a single location, on the buildspec.